src/ci/docker/x86_64-gnu-tools/checktools.sh

```diff
@@ -59,6 +77,11 @@ if [ "$RUST_RELEASE_CHANNEL" = nightly -a -n "${TOOLSTATE_REPO_ACCESS_TOKEN+is_s
         sed -i "1 a\\
 $COMMIT\t$(cat "$TOOLSTATE_FILE")
 " "history/$OS.tsv"
+    # if we are at the last week in the 6-week release cycle, reject any kind of regression.
+    if [ $SIX_WEEK_CYCLE -eq 5 ]; then
```

Comment:
I would be a lot more comfortable if this was a 4, since the 5 basically doesn't help much -- last release, we landed breakage to RLS on Monday of the 2nd to last week, which as far as I can tell, this wouldn't have caught. I would like the tools to ideally be green the week of the release, and the week before the release. That seems like a fairly reasonable requirement.

Reply:
That is 1/3 rd of the release cycle, which seems kind of long. I think what is important is for tool authors to know if their tool is broken going into that final week, since they probably don;t keep track of the release cycle.

Reply:
I'm fine with leaving it at 5 for a cycle or two and then reevaluating if necessary; my hope is that over time there will be some other solution that we come up with...

Reply:
What about T-10 days as a compromise? 🤔